### PR TITLE
Udemy (Independent Publisher)

### PR DESCRIPTION
**Pull request by nanddeepn**
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding.

image

image

image

image
 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
image

- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 
Connector uses Basic Auth, not OAuth.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Hi @nanddeepn Thank you so much for submitting a connector to the Power Platform! We will review your submission within the next 2 weeks.

File: independent-publisher-connectors/Udemy/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+                              "type": "string",
+                              "description": "initials"
+                            },
+                            "url": {
+                              "type": "string",
+                              "description": "url"
+                            }
+                          }
+                        },
+                        "description": "visible_instructors"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Udemy/apiDefinition.swagger.json:123]: @nanddeepn All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: independent-publisher-connectors/Udemy/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#a435f0",
+    "capabilities": [],
+    "publisher": "Nanddeep Nachan, Smita Nachan"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Udemy/apiProperties.json:33]: @nanddeepn Please add the stackOwner property.

https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@nanddeepn Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Comment by nanddeepn** [comment]
Thank you @natalie-pienkowska, @sriyen-msft, and everyone for your guidance. 🙏

**Pull request merged**